docker-compose.yml

```diff
@@ -8,19 +8,29 @@ services:
       - "8080:8080"
     environment:
       fhir_version: 'R4'
-      hapi.fhir.server_address: 'http://localhost:8080/baseR4'
```

Comment:
Removed this because we don't have any servers or integration tests configured this way. Leaving it in causes the integration tests, when run on your local system, to fail.